content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+---
+title: API Server Flow Control
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The{{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} has some controls
+available (i.e. the `--max-requests-inflight` and
+`--max-mutating-requests-inflight` command-line flags) to limit the amount of
+outstanding work that will be accepted, preventing a flood of inbound requests
+from overloading and potentially crashing the API server, but beyond that, the
+Priority and Fairness (or flow-control) API provides a configurable mechanism to
+ensure that in a high-load situation, forward progress will be made on the right
+requests. It also allows for a limited amount of queuing, so that in the event
+of _bursty_ traffic that remains below acceptable levels on average, no requests
+need be dropped.
+
```

Comment:
It should also be said that this feature --- like the max inflight limiting --- applies only to requests that are not "longrunning".